python/samples/samplebase.py

```diff
@@ -18,6 +18,10 @@ def __init__(self, *args, **kwargs):
         self.parser.add_argument("-p", "--pwmbits", action="store", help="Bits used for PWM. Something between 1..11. Default: 11", default=11, type=int)
         self.parser.add_argument("-l", "--luminance", action="store_true", help="Don't do luminance correction (CIE1931)")
         self.parser.add_argument("-b", "--brightness", action="store", help="Sets brightness level. Default: 100. Range: 1..100", default=100, type=int)
+        self.parser.add_argument("-m", "--hardware_mapping", help="Hardware Mapping: regular, adafruit-hat, adafruit-hat-pwm" , choices=['regular', 'adafruit-hat', 'adafruit-hat-pwm'], type=str)
```

Comment:
When we have the argument parsing in the RGBMatrixOptions class, we don't need it here (also, it is confusing: are all named differently from the C++ options, which will be a bad user experience and will trigger questions on the forum. So they should be named `--led-gpio-mapping`, `--led-pwm-lsb-nanoseconds`, etc. With the `--led-` prefix and dashes instead of underscores; but anyway, that will happen automatically if we use the standard flags parsing code).